### PR TITLE
clarifying usage of apis and web apps

### PR DESCRIPTION
**Pull request by christinaausley**
Per [this Slack thread](https://camunda.slack.com/archives/C026U8GBNSW/p1652683850174159), we needed to clarify the usage of the REST APIs and web apps according to our licensing.

### CODE REVIEW

File: docs/reference/licenses.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Licenses and license information for Zeebe can be found in the Zeebe project [RE
...
+Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Platform Self-Managed Free Edition license](https://camunda.com/legal/terms/cloud-terms-and-conditions/camunda-cloud-self-managed-free-edition-terms/) or the Camunda Platform Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Platform Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Platform Self-Managed Enterprise Edition](https://camunda.com/products/cloud/camunda-cloud-enterprise-contact/). This licensing information also applies to web apps, APIs, and clients.
```
Comment (Zelldon) [inline docs/reference/licenses.md:17]: :x: Not sure whether this applies to the clients. Lets wait for @felix-mueller to clarify.

**Review by Zelldon — COMMENTED** [review]
Thanks @christinaausley looks good in general, one remark see below.

File: docs/reference/licenses.md (hunk 1 above)
Comment (felix-mueller) [reply docs/reference/licenses.md:17]: Which clients are you referring to? Operate, Tasklist, Optimize do not have clients. Identity has an internal client that is used by Operate, Tasklist, Optimize right?

Comment (christinaausley) [reply docs/reference/licenses.md:17]: @felix-mueller Yes -- so technically, we can remove clients here. Adjusted in my latest commit.

**Comment by christinaausley** [comment]
@Zelldon @felix-mueller Mind passing me an approval once more from one of you so I can merge? Thanks!

**Review by Zelldon — APPROVED** [review]

**Pull request merged**